build.py

```diff
@@ -20,19 +20,18 @@
 # load settings file
 from buildsettings import buildSettings
 
+defaultBuild = None
 # load option local settings file
-try:
+if os.path.isfile('./localbuildsettings.py'):
```

Comment:
If file is missing, there will be an exception. Not great, not terrible.

BTW, we can use:
```
# load option local settings file and default build
try:
    from localbuildsettings import buildSettings as localBuildSettings, defaultBuild
    buildSettings.update(localBuildSettings)
except ImportError:
    localBuildSettings = {}
    defaultBuild = None
```